portfolio/src/main/webapp/index.html

```diff
@@ -34,10 +34,22 @@ <h1 class="display-1 text-center my-auto">Richie Goulazian</h1>
       </div>
       <hr class="mb-4" />
 
-      <h3>About me</h3>
-      <p><i>tldr;</i> I am a computer science student currently studying at the University of Pittsburgh.</p>
-      <p>I think my most defining characteristic is that I am a big fan of learning by doing. Over the past four years, I have been
-      learning computer science through personal projects that span a lot of different disciplines.</p>
+      <div class="row">
+        <div class="col-md-9">
+          <h3>About me</h3>
+          <p><i>tldr;</i> I am a computer science student currently studying at the University of Pittsburgh.</p>
```

Comment:
nit: realized usually it's spelled tl;dr :)

Reply:
Oh wow. I guess I'm not a real denizen of the internet yet :'(. Will fix.